lib/ASN1/Base128.php

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
Really don't like having this under `FG\ASN1`. Would `FG\ASN1\Util` sound better? Or something entirely different?